start-from-zero/evaluation-zero.html

```diff
@@ -4,26 +4,27 @@
     <meta charset="UTF-8">
     <title>Evaluation</title>
     <link rel="stylesheet" href="https://necolas.github.io/normalize.css/7.0.0/normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
     
     <nav class="menu">
-        <ul>
-            <li><img src="assets/126-Softvision-logo.png"
-             alt=""></li>
-            <li class="container">
-                <div class="row-flex">
-                    <ul class='inner-ul'>
-                        <li><a href="#0">Evaluations</a></li>
-                        <li><a href="#0">New Evaluations</a></li>
-                    </ul>
-                </div>
-            </li>
-            
-            <li class='logout'>Logout</li>
-            
-            
+        
+         <div class="container" id='height-100'>
```

Comment:
I wanted to increase the specificity on those 3 containers so I could add a height of 100% on all of them. It was pretty dirty, I already fixed that.